package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -241,8 +241,10 @@ public class GeometrySubsets {
                              SceneImportOptions options) {
       var mf = ImporterBase.GetOrAddComponent<MeshFilter>(go);
       var mr = ImporterBase.GetOrAddComponent<MeshRenderer>(go);
-      if (mf.sharedMesh == null) {
-        mf.sharedMesh = new Mesh();
+      if (mf.sharedMesh == null)
+      {
+        mf.sharedMesh = new Mesh {name = Guid.NewGuid().ToString()};
+        mf.sharedMesh.MarkDynamic();
```

Comment:
Ideally, this would only be set if the mesh were animated in the USD file, given how many static meshes a USD set can have.

If not for this change, maybe just add a TODO comment here?

Reply:
Done. Here and in the buildSkinnedMesh.